release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -453,6 +453,18 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/using-an-egress-router-ovn.adoc[Considerations for the use of an egress router pod].
 
+[id="ocp-4-8-ingress-controller-power-of-two-random-choices"]
+==== Power of Two Random Choices balancing algorithm for Ingress Controller
+
+{product-title} Ingress Controllers now uses the link:https://www.haproxy.com/blog/power-of-two-load-balancing/[Power of Two Random Choices] balancing algorithm for more even balancing after reloading. The Power of Two Random Choices balancing algorithm is set by default by the `random` parameter for `haproxy.router.openshift.io/balance`. See xref:../networking/routes/route-configuration.adoc#nw-route-specific-annotations_route-configuration[Route-specific annotations] for more information.
+
+To revert the Ingress Controller back to use the Least Corrections balancing algorithm, you can use the following `oc patch` command to specify `leastconn` for `haproxy.router.openshift.io/balance`:
+
+[source,terminal]
+----
+$ oc -n openshift-ingress patch ingresses/default --type=merge --patch='{"spec":{"unsupportedConfigOverrides":"set-default-balancing-algorithm-to-leastconn"}}'
```

Comment:
The command needs to be updated:
```suggestion
$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge --patch='{"spec":{"unsupportedConfigOverrides":{"loadBalancingAlgorithm":"leastconn"}}}'
```
I realize it came from the enhancement document; sorry about that.  I'll update the enhancement with the correct command.

Reply:
Thank you!